helm-chart/renku-gateway/templates/secret.yaml

```
@@ -11,3 +11,6 @@ type: Opaque
 data:
   gitlabSudoToken: {{ .Values.gitlabSudoToken | default .Values.global.gitlab.sudoToken | b64enc | quote }}
   oidcClientSecret: {{ .Values.oidcClientSecret | default .Values.global.gateway.clientSecret | b64enc | quote }}
+  gitlabClientSecret: {{ .Values.gitlabClientSecret | default .Values.global.gateway.gitlabClientSecret | b64enc | quote }}
+  jupyterhubClientSecret: {{ required "Fill in .Values.jupyterhub.clientSecret with `openssl rand -hex 32`"  .Values.jupyterhub.clientSecret | b64enc | quote }}
+  gatewaySecret: {{ .Values.secretKey | b64enc | quote }}
```

add same guard as above
```suggestion
  gatewaySecret: {{ required "Fill in .Values.secretKey with `openssl rand -hex 32`"  .Values.secretKey | b64enc | quote }}
```